ibm/service/kubernetes/resource_ibm_container_vpc_cluster_test.go

```diff
@@ -358,6 +368,7 @@ resource "ibm_container_vpc_cluster" "cluster" {
 	vpc_id            = ibm_is_vpc.vpc.id
 	flavor            = "cx2.2x4"
 	worker_count      = 1
+	kube_version      = "1.30"
```

Comment:
IKS 1.30 is not yet GA, is it okay here? or this feature works only on 1.30?

Reply:
what happens if end user provide this parameter for IKS version less than 1.30 or ROKS 4.15?

Reply:
The API will throw back an error message, that this field is not supported on non secure-by-default clusters. I'll add this restriction to the option flag description in the doc.